addons/website_blog/tests/test_website_blog_flow.py

```diff
@@ -58,3 +61,29 @@ def test_website_blog_followers(self):
         self.assertIn(
             self.user_employee.partner_id, test_blog_post.message_partner_ids,
             'website_blog: people commenting a post should follow it afterwards')
+
+    @users('portal_user')
+    def test_blog_comment(self):
+        """Test comment on blog post with attachment."""
+        test_blog = self.env['blog.blog'].sudo().create({
+            'name': 'New Blog',
+        })
+        test_blog_post = self.env['blog.post'].sudo().create({
+            'name': 'New Post',
+            'blog_id': test_blog.id,
+            'website_published': True,
+        })
+
+        attachment = self.env['ir.attachment'].sudo().create({
+            'name': 'some_attachment.pdf',
+            'res_model': 'mail.compose.message',
+            'datas': 'test',
+            'type': 'binary',
+            'access_token': 'azerty',
+        })
+
+        with MockRequest(self.env):
+            PortalChatter().portal_chatter_post('blog.post', test_blog_post.id, 'Test message blog post', attachment_ids=str(attachment.id), attachment_tokens=attachment.access_token)
```

Comment:
Could be great to test that giving a wrong access token crashes, just to ensure it is still protected.?

Reply:
Good idea :)